pkg/kubelet/volume/empty_dir/empty_dir_test.go

```diff
@@ -27,7 +27,7 @@ import (
 
 func TestCanSupport(t *testing.T) {
 	plugMgr := volume.PluginMgr{}
-	plugMgr.InitPlugins(ProbeVolumePlugins(), &volume.FakeHost{"/tmp/fake"})
+	plugMgr.InitPlugins(ProbeVolumePlugins(), &volume.FakeHost{"/tmp/fake", nil})
```

Comment:
@erictune I remember why now; if you use field initializers you have to initialize all the fields.  If you just do:

```go
volume.FakeHost{"/tmp/fake"}
```

you get a too-few fields in initializer error.  If you don't want to supply the `nil`, you have to do:

```go
volume.FakeHost{RootDir: "/tmp/fake"}
```

I'm open to changing it or making a factory method.